### ISSUE TITLE
Couldn't upgrade the old database to GLPI 9.2

### ISSUE THREAD

**Issue by zawaye** [glpi-project/glpi#2847]
Dear GLPI Team,

I can't upgrade the database of GLPI 9.1.3 to GLPI 9.2. The previous version can easily upgrade by replacing glpi folder by overriding new version. 

Thanks & Regards,
Zaw Aye

**Comment by trasher** [comment]
Please provide extra informations.

**Comment by zawaye** [comment]
When I try to upgrade, I got the following error message:

> **Api

 users tokens has been reset, if you use REST/XMLRPC api with personal token for authentication, please reset your user's 

> token.**

> **_9.2

 multiple alter in glpi_states - Error during the database query: ALTER TABLE `glpi_states` ADD UNIQUE `unicity` (`states_id`, `name`) , ADD INDEX `is_visible_line` (`is_visible_line`) - Error is Duplicate entry '0-Active' for key 

> 'unicity_**

I have made upgrade process smoothly form 9.1.5 to 9.1.6. But, I got above error message when I try to upgrade from 9.1.6 to 9.2. I couldn't still find how to solve this problem. Could you please instruct me how to fix this problem?

**Comment by trasher** [comment]
You have duplicate state name on a same level; this was permitted before, but should not.

Rename duplicate states, and then you can upgrade.

**Comment by rjmontenegro** [comment]
Thanks.
Solved.

**Comment by zawaye** [comment]
Thanks, Trasher. But I couldn't find duplicate state. Could you kindly point me where can I find the duplicate state? I really want to use 9.2 and really excited to use on its feature.
Thanks for sharing.

**Comment by trasher** [comment]
@zawaye you'll find the list in "Setup", "Dropdowns", "Statuses of items". Each name must be unique on the same level (same parent).

**Comment by zawaye** [comment]
Thanks a lot, trasher. I can upgrade now. But, I couldn't see the previous created tickets and current created ticket under the ticket menu although I made search with status = all. 
But, I can see in the linked ticket to link in new ticket creation. 
I really appreciate for your help. 

Thanks again!

**Comment by trasher** [comment]
Cannot help on this, I do not understand the issue. Nothing similar has been reported right now; but if this is a usage issue; you'd better use forums.

**Comment by zawaye** [comment]
Thanks, Trasher

**Comment by Miminoux** [comment]
same issue (dupplicate item status), thx

**Comment by jctcom** [comment]
Can someone please explain in layman's terms how to fix this issue on my Synology NAS Box?  I cannot log into the GLPI because I get prompted to update the database upon attempting to access the site which results in this error.  I am guessing I have to do something in PHPAdmin or something?  But what exactly?

Thanks.

Carl

**Comment by trasher** [comment]
@jctcom you have te review your status to find duplicates (same name, same level) and fix them before the upgrade. This can be achieved in the old GLPI before upgrading.
